src/tools/race/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
 find_package(Threads)
+find_package(LibRt)
 
 if (APPLE)
 	remove_tool(race "race tool does not compile under MAC OS X")
 else()
+	if (NOT LIBRT_FOUND)
+		remove_tool(race "race tool requires librt support")
```

Comment:
You are right about OpenBSD providing some of librt features. Compiling and running `race` in OpenBSD without librt does not cause problems.

On Linux these symbols are provided by librt:

    /elektra/src/tools/race/race.c:102: undefined reference to `shm_unlink'
    /elektra/src/tools/race/race.c:104: undefined reference to `shm_open'
    /elektra/src/tools/race/race.c:120: undefined reference to `shm_unlink'
    /elektra/src/tools/race/race.c:178: undefined reference to `shm_unlink'